packages/tools/examples/splineContourSegmentationTools/index.ts

```diff
@@ -205,14 +207,19 @@ addDropdownToToolbar({
     const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
 
     // Set the old tool passive
-    toolGroup.setToolPassive(selectedToolName);
+    toolGroup.setToolPassive(selectedToolName, { removeAllBindings: true });
```

Comment:
I hate this, we should not have actually made it default, just remove it from the default that adds it

Reply:
What do you mean by that? Remove the `options` parameter and change `setToolPassive` to remove all bindings?